src/main/java/com/google/alpollo/TextToSpeechService.java

```diff
@@ -0,0 +1,70 @@
+package com.google.alpollo;
+
+import com.google.api.gax.rpc.FixedHeaderProvider;
+import com.google.cloud.texttospeech.v1.*;
+import com.google.protobuf.ByteString;
+import java.io.IOException;
+
+/**
+ * Contains methods for converting text to speech.
+ */
+public class TextToSpeechService {
+  private static TextToSpeechSettings settings;
+  private static final String defaultLanguageCode = "en-US";
```

Comment:
Constant fields should have their name fully capitalized

Reply:
forgot sometimes :(